site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/SupportForm.js

```diff
@@ -165,7 +165,7 @@ class SupportForm extends Component {
               });
             },
             maxSize: maxFileSize,
-            accept: ".jpeg,.jpg,.png",
+            accept: ".jpeg,.jpg,.png,.pdf",
```

Comment:
```suggestion
```

TBH, I'm not sure why we are limiting the accepted file types. I would just accept anything (imagine e.g. a user wanting to replace a `README.md` file).